src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -33,14 +34,18 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        User user = userService.findByEmail(authentication.getName()).orElseThrow(
+                () -> new RuntimeException("Invalid email"));
```

Comment:
```suggestion
                () -> new NSEE("Invalid email"));
```
do not forget about it

Reply:
change everywhere